tools/common/Optimizations.cs

```diff
@@ -0,0 +1,130 @@
+using System.Linq;
+
+namespace Xamarin.Bundler {
+	public class Optimizations
+	{
+		static string [] opt_names =
+		{
+			"remove-uithread-checks",
+			"dead-code-elimination",
+			"inline-isdirectbinding",
+			"inline-intptr-size",
+#if MONOTOUCH
+			"inline-runtime-arch",
+#endif
```

Comment:
it can be added in a separate PR but this is missing other optimizations: method/type sealer, devirtualizer and metadata removal (currently mostly parameter names). Except the last those are not _safe_ today with XM (unless we add an option of enforce the lack of SRE/code loading).

Reply:
@spouliot Yes, I wanted to not make this change *too* big, it's already big enough.